README.md

```diff
@@ -69,6 +69,52 @@ options for you and run the image using `podman`:
     ~/src/flathub/com.example.App/com.example.App.json
 ```
 
+### On Flathub
+
+Flathub runs this tool hourly for all Flatpak repos under [github.com/flathub](https://github.com/flathub). 
+This means for those repos to receive update PRs, simply ensure any `x-checker-data` you need is present [as described below](https://github.com/flathub/flatpak-external-data-checker#changes-to-flatpak-manifests). 
+Note Flathub's hosted tool only checks the default branch.
+
+To stop Flathub's tool from checking your repo, add `"disable-external-data-checker": true` to `flathub.json` in the default branch.
+
+### Custom workflow
+
+Alternatively, you can use own workflow. This can be useful if e.g. wanting to update non-default branches.
+
+Put this yaml file under `.github/workflows`, e.g. put it in `.github/workflows/update.yaml`. Ensure to put the correct path to the manifest in the last line.
+
+```yaml
+name: Check for updates
+on:
+  schedule: # for scheduling to work this file must be in the default branch
+  - cron: "0 * * * *" # run every hour
+  workflow_dispatch: # can be manually dispatched under GitHub's "Actions" tab 
+
+jobs:
+  flatpak-external-data-checker:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        branch: [ master ] # list all branches to check
+    
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ matrix.branch }}
+
+      - uses: docker://ghcr.io/flathub/flatpak-external-data-checker:latest
+        env:
+          GIT_AUTHOR_NAME: Flatpak External Data Checker
+          GIT_COMMITTER_NAME: Flatpak External Data Checker
+          GIT_AUTHOR_EMAIL: 41898282+github-actions[bot]@users.noreply.github.com
```

Comment:
Where does this magic number come from?

Reply:
Indeed it might.

Could you add a comment explaining this?

Reply:
> Could you add a comment explaining this?

done